CHANGELOG.md

```diff
@@ -48,7 +48,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Issue 3476: Traffic Router returns partial result for CLIENT_STEERING Delivery Services when Regional Geoblocking or Anonymous Blocking is enabled.
 - Upgraded Traffic Portal to AngularJS 1.7.8
 - Issue 3275: Improved the snapshot diff performance and experience.
-
+- Issue 3605: Fixed Traffic Monitor custom ports in health polling URL.
```

Comment:
nit: since this is a GitHub markdown file, if you precede the issue number with <kbd>#</kbd> it'll create a link. Which is neat.

Reply:
Done